main.py

```diff
@@ -2,8 +2,12 @@
 from src.image_generation.image_generator import ImageGenerator, create_image_generator, download_and_save_image
 from src.gpt.text_generator import request_chat_completion
 import logging
+from src.fine_tuning.fine_tuning_job import upload_training_file
+from src.fine_tuning.fine_tuning_job import create_fine_tuning_job
 
-
+# response = upload_training_file("propaganda.jsonl")
```

Comment:
Should be removed as we do not want to start a new finetuning job for each run of the application